src/condition.js

```diff
@@ -101,9 +102,9 @@ export default class Condition {
     ]).then(([rightHandSideValue, leftHandSideValue]) => {
       const result = op.evaluate(leftHandSideValue, rightHandSideValue)
       debug(
-        `condition::evaluate <${JSON.stringify(leftHandSideValue)} ${
+        `condition::evaluate <${jsonStringifyWithBigInt(leftHandSideValue)} ${
           this.operator
-        } ${JSON.stringify(rightHandSideValue)}?> (${result})`
+        } ${jsonStringifyWithBigInt(rightHandSideValue)}?> (${result})`
```

Comment:
Let's avoid a custom stringifier here when we can instead rely on the `console.log` built-in formatter if we change the signature of `debug`